src/output/plugins/OSXOutputPlugin.cxx

```diff
@@ -914,6 +925,7 @@ OSXOutput::Play(const void *chunk, size_t size)
 			throw std::runtime_error("Unable to restart audio output after pause");
 		}
 	}
+	while (cancel);
```

Comment:
Another busy loop.

Reply:
This one would be changed to a condition variable wait. But we will need to do a lot of changes before that happens (Such as silence generation, and other changes you proposed). Better not to do it in this PR

Again it only happens during Cancel, so the impact is very minimal.